analysis/topEFT/work_queue_run.py

```diff
@@ -166,8 +166,8 @@ def LoadJsonToSampleName(jsonFile, prefix):
 
 executor_args = {
     'master_name': '{}-workqueue-coffea'.format(os.environ['USER']),
-
-    'xrootdtimeout': 180,
+    'filepath': '/tmp', 
+    'xrootdtimeout': 300,
```

Comment:
I know these are typical changes we've been using, but do we want to push them? @kmohrman what do you think?